azure-pipelines.yml

```diff
@@ -190,6 +190,7 @@ jobs:
 
       - script: |
           cd packaging/iofog-agent
+          sed -i 's/default/dev/g'  etc/iofog-agent/config-switcher_new.xml
```

Comment:
Does this work on mac and linux? In my experience need `-i.bkp` for linux. Need to specify backup file suffix

Reply:
yeah this won't work. `sed -i '' 's/default/dev/g' etc/iofog-agent/config-switcher_new.xml` should be used or `-i.bkp`. Have used `-i.bkp` now